packages/material-ui/package.json

```diff
@@ -48,6 +48,7 @@
     "hoist-non-react-statics": "^3.2.1",
     "is-plain-object": "^2.0.4",
     "normalize-scroll-left": "^0.1.2",
+    "object-is": "^1.0.1",
```

Comment:
Do we need a dependency for such a small module?
https://github.com/ljharb/object-is/blob/master/index.js